test/library/packages/DistributedDeque/DequeInterleavedLifoIter.future

```diff
@@ -0,0 +1 @@
+bug: Certain insertion patterns make LIFO iteration crash (#27505)
```

Comment:
We typically have this on separate lines.  From [the testing system documentation](https://chapel-lang.org/docs/developer/bestPractices/TestSystem.html#a-test-that-tracks-a-failure) (emphasis mine):

> The format of the .future file itself is minimally structured. The **_first line_** should contain the type of future (see list below) followed by a brief (one 80-column line) description of the future, which ideally reflects the associated GitHub issue title. The **_next line_** should contain the associated GitHub issue number in the #issue-number format, e.g. #1.